docker-compose.yml

```diff
@@ -1,14 +1,15 @@
-version: '2'
+version: '2.3'
```

Comment:
you should leave the original docker-compose here

Reply:
If I am not mistaken, if you wanna use `nvidia` as `runtime`, then the version must be 2.3, no?

Reply:
we can change the versions but all other attributes should be the same as before (or at least not point to your folders)

Reply:
Yup, I finally reverted back to the original form... so this shouldnt be a problem anymore